config_defaults.yml

```diff
@@ -78,6 +79,13 @@ Gemfile:
       - gem: master_manipulator
       - gem: beaker-hostgenerator
         from_env: 'BEAKER_HOSTGENERATOR_VERSION'
+
+MAINTAINERS:
+  maintainers:
+    - github: puppetlabs
+      email: "discuss-modules@puppet.com"
```

Comment:
It's a gmail backed list that probably doesn't even allow unmoderated
posting, that already (by this PR) has been published. (shrug)

On 14 September 2016 at 16:57, Glenn Sarti <notifications@github.com> wrote:

> In config_defaults.yml
> <https://github.com/puppetlabs/modulesync_configs/pull/92#discussion_r78778478>
> :
>
> > @@ -78,6 +79,13 @@ Gemfile:
> >        - gem: master_manipulator
> >        - gem: beaker-hostgenerator
> >          from_env: 'BEAKER_HOSTGENERATOR_VERSION'
> > +
> > +MAINTAINERS:
> > +  maintainers:
> > +    - github: puppetlabs
> > +      email: "discuss-modules@puppet.com"
>
> @ferventcoder <https://github.com/ferventcoder> has mentioned previously
> we should obsfucate the email to reduce spamming that account
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/puppetlabs/modulesync_configs/pull/92/files/d6520992c0fabec2c41c07356883e1833db7f79f#r78778478>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABsQDRz0FJX2vKfIjIw1euspdYc1Ifkks5qqBlRgaJpZM4J1nlk>
> .
>